Cargo.toml

```diff
@@ -110,6 +110,8 @@ object.debug = 0
 rustc-std-workspace-core = { path = 'library/rustc-std-workspace-core' }
 rustc-std-workspace-alloc = { path = 'library/rustc-std-workspace-alloc' }
 rustc-std-workspace-std = { path = 'library/rustc-std-workspace-std' }
+object = { git = 'https://github.com/gimli-rs/object.git' }
```

Comment:
Blocking PR: Arm64EC support has been merged into the `object` crate, but has not been released yet.